lib/biokbase/catalog/registrar.py

```diff
@@ -437,15 +434,15 @@ def validate_method_specs(self, basedir):
                         display_yaml = display_yaml_file.read()
 
                     # gather any extra html files
-                    extraFiles = {}
+                    extra_files = {}
```

Comment:
If this really is dead code it should be deleted. If it's not dead code and we're changing it we should test it.

Reply:
this section is an argument for Java :P

Reply:
So, this looks to me to support adding additional html files to a method for extra documentation. We probably never used the functionality so never noticed it was broken. I can test by adding an extra html file to https://github.com/kbaseIncubator/catalog_test_module/tree/simple_good_repo/ui/narrative/methods/test_method_1
But I'm not sure it's even worth the effort.

Reply:
Hmm, I could've sworn that some modules had extra html files. Maybe it worked at one point?

If you want to remove it, we should check whether a) any modules have extra html files and b) whether we still need to support the feature.

Reply:
So I added a trivial test that registers a module with extra html files just to make sure that nothing breaks. I can't really figure out what NMS does with the extra files but I'll argue that those tests belong in NMS not here.